server/src/main/java/org/apache/druid/segment/loading/LeastBytesUsedStorageLocationSelectorStrategy.java

```diff
@@ -32,11 +34,17 @@
 public class LeastBytesUsedStorageLocationSelectorStrategy implements StorageLocationSelectorStrategy
 {
   private static final Ordering<StorageLocation> ORDERING = Ordering.from(Comparator
-      .comparingLong(StorageLocation::currSizeBytes));
+                                                                              .comparingLong(StorageLocation::currSizeBytes));
```

Comment:
Please roll back useless code formatting.

Reply:
It's accidentally formatted by IDEA. Been fixed.